src/index.js

```diff
@@ -4,9 +4,10 @@ const {promisify} = require('bluebird');
 const nopt = require('nopt');
 const npm = require('npm');
 const request = require('request-promise').defaults({json: true});
-const getLog = require('./lib/log');
 const ownPkg = require('../package.json');
-let pkg = JSON.parse(readFileSync('./package.json'));
+const getLog = require('./lib/log');
+
+const pkg = JSON.parse(readFileSync('./package.json'));
```

Comment:
this file currently fails for me. Do you see any reason why we do `JSON.parse(readFileSync())` here instead of just `require`?

Reply:
yeah let’s change this line to `require('../package.json')` then it all works for me

Reply:
let me do it actually :)

Reply:
That's really weird. I just changed the order to group the import together.

No idea why we use `JSON.parse(readFileSync()` instead of `require('../package.json')`.
By the way it should be `require('./package.json')` not `require('../package.json')`.

Maybe you are not executing from the root of your repo?

Reply:
Actually I know why. `JSON.parse(readFileSync('./package.json'));` use path relative to `cwd` so it import the user's repo `package.json`. `require` use  a path relative to `_dirname` so that would import the `package.json` of the cli

Reply:
we should probably do something like `require(path.resolve(process.cwd(), 'package.json'))` so we don’t need the JSON parsing. But I guess it will be the same at the end of the day, I’mm gonna merge this as  is :)